src/terra-dev-site/test/loaders/pretendPackage.json

```diff
@@ -1,4 +1,7 @@
 {
   "name": "terra-dev-site",
-  "version": "6.2.0"
+  "version": "6.2.0",
+  "src": {
```

Comment:
i think i prefer `source` over `src`, anyone else have any opinions?

Reply:
What are your thoughts on changing this to be the `homepage` key in the package.json? This is a standardized npm key and it would also be picked up on npmjs.com (though maybe these should be updated to be terra-ui/components/compenent_name?)

![Screen Shot 2020-04-01 at 1 37 02 PM](https://user-images.githubusercontent.com/14099737/78173826-e995be80-741d-11ea-9ad8-91cd8a0c6c5e.png)


In non-mono repo projects, this moves them to the repo.

In mono-repo projects (once we update the homepage urls), they will direct them to the package in the mono-repo....Plus it'd finally give these package-level READMEs some purpose 😄

Reply:
I do think since this is a custom package.json key, we dont' need the additional nesting for URL. We could simplify to:
```
source: "url"
```

That being said, are we sure `src` or `source` doesn't conflict with existing build tools? This might be too generic and used by something else we might not realize

Reply:
Homepage should take you to terra-ui from the opensource npm repo. I thought about re-using it but it didn't make sense. Another though i had was to try to convert the repo to a url, but i don't think that sounds easy to get right.
Maybe add a terra-dev-site section?
```json
"terra-dev-site": {
    "source": "source string"
}
```

Reply:
Updated the custom property to be as follows :
```
"terra-dev-site": {
    "source": "source string"
}
```
Changed here : https://github.com/cerner/terra-dev-site/pull/243/commits/6dde4584e4c4d54343e4b4f2efdf3f0be133ebf7

Reply:
so for the monorepo's loaders will be reading from package.json  correct ? so will the custom property added there in the same pattern as above ?  or just a thought like are we considering adding it under the [respository](https://github.com/cerner/terra-core/blob/master/package.json#L9) section ?

` "repository": {
    "type": "git",
    "url": "git+https://github.com/cerner/terra-core.git"
    "source": "source string"
  },`

Reply:
I'd like to avoid modifying existing package structures to avoid future conflicts with the package structure.

Reply:
I've gone back and forth on deciding on if it like this approach or not. @mjhenkes and I were messaging back & forth & looking at other mono-repos like Babel, Angular, etc to see if there is an industry standard for defining these. It looks like NPM has a recommendation for this:
https://docs.npmjs.com/files/package.json#repository

> If the package.json for your package is not in the root directory (for example if it is part of a monorepo), you can specify the directory in which it lives:
```json
"repository": {
  "type" : "git",
  "url" : "https://github.com/facebook/react.git",
  "directory": "packages/react-dom"
}
```
Let's go with this approach. For mono-repos we can build the url if directory exists, otherwise we will just use repository.url.

Reply:
+1, thanks @emilyrohrbough  for finding that in the doc.

Reply:
Changed here : https://github.com/cerner/terra-dev-site/pull/243/commits/c21ace7dde38d17d312bfc552ddc96765b46b6ab